be/src/server.js

```diff
@@ -9,31 +10,32 @@ const fastify = Fastify({
 
 const { PORT } = process.env || 3000;
 
-fastify.put('/', (request, reply) => {
-	collection.insertOne(request.body)
-	reply.send('Insertion succeeded')
-})
+fastify.post('/*', async (request, reply) => {
+
+	const input = JSON.stringify(request.url) + JSON.stringify(request.body)
+	const hashed = hash(input)
 
-const getIdSchema = {
-	params: {
-		type: 'object',
-		properties: {
-			id: { type: 'number' }
-		}
+	const doc = await collection.findOne({ _id: hashed })
+
+	if (doc) {
+		return reply.send({
+			...doc,
+			mongo: true
+		})
```

Comment:
missing `;` here and in other places

Reply:
I know :) that's because we're keeping it as in svizzle where changing hat rule would be a massive PR and we postponed, for now I'd use them everywhere (sorry!)

Reply:
we'll get there asap https://github.com/nestauk/svizzle/issues/510